architecture/FORM_OPTIONS_WRAPPERS.md

```
@@ -0,0 +1,993 @@
+  'btn-group': props.buttons,
+  [`btn-group-${size.value}`]: size.value !== 'md' && props.buttons,
+  'is-valid': state.value === true,
+  'is-invalid': state.value === false,
+}))
+```
+
+## Proposed Solution: Base/Wrapper Pattern
+
+### Architecture
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)